### ISSUE TITLE
Change all folders, DB name etc. from to "eventyay"

### ISSUE THREAD

**Issue by mariobehling** [fossasia/eventyay-talk#24]
### Problem and impact

Change all folders, DB name etc. from "pretalx" to "eventyay"

**Comment by Kaushik-Iyer** [comment]
can I take this up?

**Comment by mariobehling** [comment]
Please go ahead. No need to claim issues. Just make a PR when you are ready.

**Comment by Kaushik-Iyer** [comment]
Do i replace every occurrence of pretalx anywhere in the whole codebase?

**Comment by mariobehling** [comment]
Yes, correct, but you need to ensure the system still works.

**Comment by mariobehling** [comment]
You can also solve this issue in two steps
1. Change the UI and text 
2. Change the code references

**Comment by AviGawande** [comment]
hello @mariobehling  is this issue still open

**Comment by mariobehling** [comment]
Yes, it is only partly solved, but I recommend you focus on other issues first as this is not the highest priority right now.
